modules/sfm/samples/data/images/dataset_files.txt

```diff
@@ -0,0 +1,4 @@
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2889.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2890.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2891.jpg
+/home/eriba/software/opencv_contrib/modules/sfm/samples/data/images/resized_IMG_2892.jpg
```

Comment:
It will be great to use relative path from this "dataset_file":
1) `<some path>/dataset_files.txt`
2) extract work dir (truncate tail after last slash: `/`) --> `<some path>/` or `""`

Contents:
```
resized_IMG_2889.jpg
resized_IMG_2890.jpg
resized_IMG_2891.jpg
resized_IMG_2892.jpg
```